.github/workflows/hive-nightly.yml

```diff
@@ -14,17 +14,10 @@ jobs:
         with:
           fetch-depth: 0 # fetch git tags for "git describe"
 
-      - name: build erigon image
```

Comment:
so we're no longer creating a new image, which gets picked by "run hive and parse output" job? where is the image for this job coming from then?